### DISCUSSION TITLE
Multiple releases with same version number?

### DISCUSSION THREAD

**Discussion by mistertheloaf** [valinet/ExplorerPatcher#3483] [General]
Today there was a release of EP with version number 22621.3810.66.1

But, last week there was a release with that same version number.

In May/June  there were 4 releases all labeled 22621.3527.65.5

Shouldn't each release have its own version number?

**Comment by pyrates999** [comment]
They are pre-releases.  If you go here, you'll see them labeled as such: https://github.com/valinet/ExplorerPatcher/releases

example:

![image](https://github.com/user-attachments/assets/04652dc6-9643-4837-a62c-110da2659344)

**Comment by Amrsatrio** [comment]
Adding to Pyrates' comment, the use of pre-releases is to spot bugs and other inconveniences before they become a release.

Pre-release builds are created immediately as soon as a new commit has arrived in this repo, meaning like, if me or Valinet has worked on a new feature or a fix and pushing it here, a new build containing those will be made.

A pre-release version stable enough (and not having a lot of VT detections) may get promoted into a release. Don't worry, I try my best not to make actual releases with a conflicting version number.

**Reply by mistertheloaf** [reply]
Yes, I understand that these are "pre-release", but all software has "pre-release" or "beta" versions.  I've never seen anyone use the same version number for multiple releases.  It just seems weird and confusing.

But, Explorer Patcher is a great program and without it Windows 11 is almost unusable, so THANK YOU for the work you do on it.

**Reply by pyrates999** [reply]
He just explained to you that he doesn't use the same version number for multiple releases.
